pydal/adapters/base.py

```diff
@@ -246,11 +243,12 @@ def _parse(self, row, tmps, fields, colnames, blob_decode,
                 new_column_name = self._regex_select_as_parser(colname)
                 if new_column_name is not None:
                     column_name = new_column_name.groups(0)
-                    setattr(new_row, column_name[0], value)
```

Comment:
@mdipierro yes, was legacy code before the refactoring of the Storage class inside pydal (happened long time ago by @ilvalle), running `setattr()` on `BasicStorage` objects is the same of setting a in item.